portal/src/locale-data/en.json

```diff
@@ -700,9 +700,9 @@
   "AddSingleSignOnConfigurationScreen.card.facebook.description": "Allow your users to Sign in with their Facebook account",
   "AddSingleSignOnConfigurationScreen.card.github.description": "Allow your users to Sign in with their GitHub account",
   "AddSingleSignOnConfigurationScreen.card.linkedin.description": "Allow your users to Sign in with their LinkedIn account",
-  "AddSingleSignOnConfigurationScreen.card.azureadv2.description": "Allow your users to Sign in with their Microsoft Azure AD",
+  "AddSingleSignOnConfigurationScreen.card.azureadv2.description": "Allow your users to Sign in with their Microsoft Entra ID",
   "AddSingleSignOnConfigurationScreen.card.azureadb2c.description": "Allow your users to Sign in with their Microsoft Azure AD B2C",
-  "AddSingleSignOnConfigurationScreen.card.adfs.description": "Allow your users to Sign in with their Microsoft Azure AD FS",
+  "AddSingleSignOnConfigurationScreen.card.adfs.description": "Allow your users to Sign in with their Microsoft Entra ID FS",
```

Comment:
This is incorrect. The original copy is also wrong. It should be `Microsoft AD FS`